NGitLab.Mock/Config/GitLabHelpers.cs

```diff
@@ -0,0 +1,1279 @@
+        /// Add label in merge request (create it if not exists)
+        /// </summary>
+        /// <param name="mergeRequest">Merge request.</param>
+        /// <param name="label">Label name (required)</param>
+        public static GitLabMergeRequest WithLabel(this GitLabMergeRequest mergeRequest, string label)
+        {
+            return Configure(mergeRequest, _ =>
+            {
+                mergeRequest.Labels.Add(label ?? throw new ArgumentNullException(nameof(label)));
+            });
+        }
+
+        /// <summary>
+        /// Add approver in merge request
+        /// </summary>
+        /// <param name="mergeRequest">Merge request.</param>
+        /// <param name="approver">Approver username (required)</param>
+        public static GitLabMergeRequest WithApprover(this GitLabMergeRequest mergeRequest, string approver)
+        {
+            return Configure(mergeRequest, _ =>
+            {
+                mergeRequest.Approvers.Add(approver ?? throw new ArgumentNullException(nameof(approver)));
+            });
+        }
+
+        /// <summary>
+        /// Add user permission in group
+        /// </summary>
+        /// <param name="group">Group.</param>
+        /// <param name="user">Username (required)</param>
+        /// <param name="level">Access level.</param>
+        public static GitLabGroup WithUserPermission(this GitLabGroup group, string user, AccessLevel level = AccessLevel.Developer)
+        {
+            return Configure(group, _ =>
+            {
+                var permission = new GitLabPermission
+                {
+                    User = user ?? throw new ArgumentNullException(nameof(user)),
+                    Level = level,
+                };
+
+                group.Permissions.Add(permission);
+            });
+        }
+
+        /// <summary>
+        /// Add user permission in project
+        /// </summary>
+        /// <param name="project">Project.</param>
+        /// <param name="user">Username (required)</param>
+        /// <param name="level">Access level.</param>
+        public static GitLabProject WithUserPermission(this GitLabProject project, string user, AccessLevel level = AccessLevel.Developer)
+        {
+            return Configure(project, _ =>
+            {
+                var permission = new GitLabPermission
+                {
+                    User = user ?? throw new ArgumentNullException(nameof(user)),
+                    Level = level,
+                };
+
+                project.Permissions.Add(permission);
+            });
+        }
+
+        /// <summary>
+        /// Add group permission in group
+        /// </summary>
+        /// <param name="grp">Group.</param>
+        /// <param name="group">Group fullname (required)</param>
+        /// <param name="level">Access level.</param>
+        public static GitLabGroup WithGroupPermission(this GitLabGroup grp, string group, AccessLevel level = AccessLevel.Developer)
```

Comment:
Is it groupName or groupId or both are accepted? I think the parameter name should be improved